### ISSUE TITLE
Documentation is unavailable

### ISSUE THREAD

**Issue by FloofyPlasma** [lumin-org/ui#9]
Currently documentation pages are trying to load content from the old package name (Aegis) resulting in broken pages. Some links need to be updated too to reflect the rename.

**Comment by 2jammers** [comment]
yep we are aware of this, the documentation will probably take a while to be completely redone, as v0.3.x is a complete rewrite and API change
